rollup.config.js

```diff
@@ -1,5 +1,11 @@
 import babel from '@rollup/plugin-babel';
 import resolve from '@rollup/plugin-node-resolve';
+import run from '@rollup/plugin-run';
+import commonJS from '@rollup/plugin-commonjs'
+import json from '@rollup/plugin-json';
+
+// equals to a flag --watch
+const isDevelopment = process.env.ROLLUP_WATCH === 'true'
```

Comment:
I am a bit concerned that we determine `isDevelopment` not by `--environment` flag or not by setting `NODE_ENV=development`. I think it's less explicit and might be confusing when we determine it by the `--watch` flag. What do you think?

Reply:
I agree w/you. It could be confusing, this is why I added the comment + maybe we will need `NODE_ENV` inside the app. I will introduce `NODE_ENV=development`